src/sync.cpp

```diff
@@ -37,23 +39,23 @@ void PrintLockContention(const char* pszName, const char* pszFile, int nLine)
 //
 
 struct CLockLocation {
-    CLockLocation(const char* pszName, const char* pszFile, int nLine, bool fTryIn)
-    {
-        mutexName = pszName;
-        sourceFile = pszFile;
-        sourceLine = nLine;
-        fTry = fTryIn;
-    }
+    CLockLocation(
+        const char* pszName, const char* pszFile, int nLine, bool fTryIn, std::string thread_name_) :
+            mutexName(pszName), sourceFile(pszFile), sourceLine(nLine), fTry(fTryIn),
```

Comment:
Reorder to match the order the compiler will use (which is the order of declaration): so please put `fTry` first :-)

Reply:
Fixed, thanks.